### PR TITLE
Re-arrange article menu/toc items

### PR DESCRIPTION
**Pull request by sharvaniharan**
**Phab:** https://phabricator.wikimedia.org/T273260

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -47,6 +47,8 @@
     <string name="menu_page_other_languages">Change language</string>
     <string name="menu_page_find_in_page">Find in article</string>
     <string name="menu_page_add_to_watchlist">Add to Watchlist</string>
+    <string name="menu_page_talk">View talk page</string>
+    <string name="menu_page_edit_history">View edit history</string>
```
Comment (cooltey) [inline app/src/main/res/values/strings.xml:51]: Do we really need the text "View" instead of showing "Talk page" and "Edit History"?

**Review by cooltey — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (sharvaniharan) [reply app/src/main/res/values/strings.xml:51]: Good catch .. looks like Jazmin has also skipped view on the ticket

Comment (sharvaniharan) [reply app/src/main/res/values/strings.xml]: I have updated it according to the strings Jazmin has put it @cooltey .

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -47,6 +47,8 @@
     <string name="menu_page_other_languages">Change language</string>
     <string name="menu_page_find_in_page">Find in article</string>
     <string name="menu_page_add_to_watchlist">Add to Watchlist</string>
+    <string name="menu_page_talk">Talk</string>
+    <string name="menu_page_edit_history">Edit History</string>
```
Comment (cooltey) [inline app/src/main/res/values/strings.xml:51]: Should we have a lowercase of `history?

Comment (sharvaniharan) [reply app/src/main/res/values/strings.xml:51]: She has kept it in Uppercase, probably like 'Add to Watchlist'

Comment (sharvaniharan) [reply app/src/main/res/values/strings.xml]: Per our conversation, I changed this to lowercase @cooltey . So done.

**Pull request merged**